index.html

```diff
@@ -86,10 +86,51 @@ <h2>Facebook 360</h2>
               <li>css</li>
               <li>javascript</li>
             </ul>
-            <form action="#">
-                <button class="btn-body btn" type="submit" name="submit">see project</button>
-            </form>
-        </div>
+            <button class="btn-body btn" id="myBtn1" type="button" name="submit">see project</button>
+          </div>
+          <div id="myModal1" class="modal">
+            <div class="modal-content">
+                <div class="modal-title">
+                  <div class="toggle-header">
+                    <div class="toggle-text-top">
+                      <h1>Facebook 360</h1>
+                      <ul class="work-top work-top-toggle">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+                        <li class="dev">back end dev</li>
+                        <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+                        <li class="dev">2015</li>
+                      </ul>
+                   </div>
+                    <span class="close1">&times;</span>
+                  </div>
+                  <img class="modal-img" src="img/Portfolio1-big.svg" alt="work-img">
+                </div>
+              
+              <div class="toggle-body">
+                <p class="toggle-p">
+                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. A labore quibusdam non optio  itaque repellat inventore nisi, magnam accusamus tenetur quod. Sint ea sit similique   incidunt explicabo voluptatibus quae quia.  Lorem ipsum dolor sit amet consectetur  adipisicing elit. Sunt, delectus laudantium laboriosam animi voluptates eveniet quis   repellat porro cupiditate officiis quibusdam doloremque libero numquam facilis unde   laborum et exercitationem nihil.
+                </p>
+                <nav class=toggle-nav>
+                 <ul class="toggle-list-top">
+                   <li> <p>html</p></li>
+                   <li><p>css</p></li>
+                   <li><p>javascript</p></li>
+                 </ul> 
+                 <ul class="toggle-list-bottom">
+                   <li><p>github</p></li>
+                   <li><p>ruby</p></li>
+                   <li><p>boostrap</p></li>
+                 </ul>
+
+                  <form action="#">
+                    <button type="button" name="submit" id="btn-toggle1" class="btn-toggle" >see live</button>
+                    <Button type="button" name="submit" id="btn-toggle2" class="btn-toggle">see source</Button>
+                  </form>
+                </nav>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Check an [exampe of students projects](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96).
- [ ] Modal popup data is populated with the data from the projects array.

Reply:
Hi Emmanuel,

I did not understand how to implement the change request you sent to me,
can we please meet on zoom to make it clearer to me.

Thank you.